src/controllers/carController.js

```diff
@@ -0,0 +1,15 @@
+const carService = require("../services/carService");
+const { asyncErrorHandler } = require("../utils/error");
+
+const getCarList = asyncErrorHandler(async (req, res) => {
+  const result = await carService.getCarList(req.query);
+  return res.status(200).json({ data: result });
+});
+
+const getSearchList = asyncErrorHandler(async (req, res) => {
+  const { keyword } = req.body;
```

Comment:
검색이라는것은 결과를 get(가져오는)하는 것입니다! 일반적으로 body에 데이터를 실어서 보내는 것은 get이 아닌 post, patch, put 등에서 사용합니다. query로 받아올 수 있게 수정해주세요
```suggestion
  const { keyword } = req.query;
```

Reply:
네 수정했습니다 !